docs/v3/chunk-grid.rst

```diff
@@ -0,0 +1,85 @@
+.. _chunk-grid-list:
+
+==========
+Chunk Grid
+==========
+
+The following core chunk grids are defined by the maintainers of
```

Comment:
are they defined by maintainers, or by the spec? seems better to leave the maintainers out of it, and just say "these are the core chunk grids"